dotCMS/src/integration-test/java/com/dotcms/MainSuite.java

```diff
@@ -316,7 +317,9 @@
         ContentletWebAPIImplIntegrationTest.class,
         Task05370AddAppsPortletToLayoutTest.class,
         FolderFactoryImplTest.class,
-        DotSamlResourceTest.class
+        DotSamlResourceTest.class,
+        FolderFactoryImplTest.class,
+        IntegrityDataGenerationJob.class
```

Comment:
This needs to be the *Test class instead of the Job itself

Reply:
You are right.